jsapp/js/components/library/assetsTableRow.es6

```diff
@@ -29,8 +24,7 @@ class AssetsTableRow extends React.Component {
     if (
       this.props.asset.asset_type !== ASSET_TYPES.collection.id &&
       this.props.asset.summary &&
-      this.props.asset.summary.row_count &&
-      this.props.asset.summary.row_count >= 2
+      this.props.asset.summary.row_count
     ) {
```

Comment:
i don't know what these other guys are talking about, but have you heard about this one weird trick called [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)?

Reply:
Alex was first, but thanks nonetheless :)

Reply:
Thanks for indulging us our (bad) joke. I'd never heard of it before today ;)